docs/develop/configuration.md

```diff
@@ -270,3 +270,37 @@ How secrets are mounted at runtime depends on the serving environment. For examp
 running on Docker locally, Truss allows specifying secrets in
 `~/.truss/config.yaml`, while on Baseten, secrets can be specified as regular
 [organization secrets](https://docs.baseten.co/settings/secrets).
+
+### hf_cache
+
+Truss supports the caching of models pulled from Hugging Face. By saving your machine learning model's weights into your image during buildtime instead of runtime, you can considerably improve cold start performance. "Cold start" refers to the period when an application is launched for the first time and has to perform all initial setup operations. In this context, having the weights preloaded means your application can start using the machine learning model faster since it no longer has to wait for the weights to download during the runtime.
```

Comment:
Just going to shorten this up a bit:

Truss supports caching models pulled from Hugging Face. By saving your model's weights into your image during buildtime instead of runtime, you can considerably reduce cold start times (the time between the first call to a model server and it reaching a ready state). In this context, having the weights preloaded means your application can start using the model faster since it doesn't have to wait for the weights to download at runtime.